src/include/kdbprivate.h

```diff
@@ -518,10 +518,13 @@ int elektraValidateKeyName(const char *name, size_t size);
  */
 #define ELEKTRA_MAX_ARRAY_SIZE (21)
 
+#if !defined(__cplusplus)
```

Comment:
are these functions only exposed in a `C` environment?

	int elektraArrayValidateName(const Key *key);
	int elektraReadArrayNumber(const char *baseName, kdb_long_long_t *oldIndex);
	int elektraWriteArrayNumber(char *newName, kdb_long_long_t newIndex);